lib/rubocop/cop/rspec/before_after_all.rb

```diff
@@ -30,11 +30,11 @@ class BeforeAfterAll < Base
               '`use_transactional_fixtures` is enabled, then records created ' \
               'in `%<hook>s` are not automatically rolled back.'
 
-        RESTRICT_ON_SEND = %i[before after].freeze
+        RESTRICT_ON_SEND = Set[:before, :after].freeze
```

Comment:
what's the reason to use a set?
Seems that for the majority of cops we use %i. If we prefer set:
* we need to update the rest of the cops
* we can provide a check in the internal investigations that RESTRICT_ON_SEND is defined as Set

Reply:
This was to make it easier to reuse in `def_node_matcher`. If we use `%i`, it would have to be something like `{:#{RESTRICT_ON_SEND.join(' :')}}`, wouldn't it?

Reply:
Ah I see. Thank you 👍